content/tutorials/getting-started/index.md

```diff
@@ -173,53 +179,41 @@ Gatling Enterprise requires a package built from a Gatling project in order to r
 
 ```
-$ ./bin/enterprisePackage.sh
-GATLING_HOME is set to .../gatling-charts-highcharts-bundle-{{< var gatlingVersion >}}
-GATLING_VERSION is set to '{{< var gatlingVersion >}}'
-Creating jar...done
+$ ./bin/gatling.sh
 ```
 
-This script can also be clicked or double clicked from any file explorer. You'll see the result in the folder `target`, which should now have a file named `package.jar`.
-
-{{< youtube _fq3giAIibw >}}
+And now type `3` to package your classes. You'll see the result in the folder `target`, which should now have a file named `package.jar`.
 
 #### Step 3.2 - Upload the Package {#upload-package}
 
 On Gatling Enterprise, click on the [Packages section]({{< ref "../../reference/user/package_conf/" >}}) and on create. Give a name to your package, you can choose to make it available to all teams or to limit access to a specific one, then hit save.
 
 Next, upload the package you generated earlier by clicking on the cloud icon. The package you generated earlier is located here: `target/package.jar`
 
-Upload it to Gatling Enterprise, either by drag-and-dropping it to the modal, or by clicking on the modal to open the file manager.
+Upload it to Gatling Enterprise, by clicking on the upload button on the right side then either drag-and-drop your file or by clicking on the modal to open the file manager. 
 
 #### Step 3.3 - Create a Simulation {#create-simulation}
 
-To get a grasp of how to run a simulation with Gatling, you will find a ready to use script in the bundle. So let's get started with your first simulation!
-
 In the [Simulations section]({{< ref "../../reference/user/simulations/" >}}), click on **Create**.
 
 Choose a name for your simulation and define the team that will have access to this simulation (if none has been created yet, a default one named **Default team** exist).
 
-Under Class name, it is important you enter the name of the script that was uploaded in the package, otherwise the run of your simulation will fail in the next step.
-
-In this case, the ready-made script called `BasicSimulation.scala` is located in the bundle: `user-files > simulations > computerdatabase`
+Then select the package you just created. 
 
-Therefore, you need to enter `computerdatabase.BasicSimulation` as the class name.
+Under Class name, select the desired simulation ClassName (here : `computerdatabase.ComputerDatabaseSimulation`) among the ones available in your package.
 
 Then, click on the **Next** button.
 
-Next, you'll need to select your package name. It is important to select the package that contains the simulation you'd like to run. Then, click on the **Next** button.
-
-In the final step, you can choose where you want the injection to take place, and click on the **Save** button.
-
+Finally, choose the location from where you want the injection to take place.
```

Comment:
Finally, choose the location where you want to spawn your load generators and generate fraffic from.